Symfony/CS/Fixer.php

```diff
@@ -46,27 +51,30 @@ class Fixer
     /**
      * ErrorsManager instance.
      *
-     * @var ErrorsManager|null
+     * @var ErrorsManager
      */
     protected $errorsManager;
 
     /**
-     * LintManager instance.
+     * Lint manager instance.
      *
-     * @var LintManager|null
+     * @var LintManagerInterface
      */
     protected $lintManager;
 
     /**
      * Stopwatch instance.
      *
-     * @var Stopwatch|null
+     * @var Stopwatch
      */
     protected $stopwatch;
 
     public function __construct()
     {
         $this->diff = new Differ();
+        $this->errorsManager = new ErrorsManager();
+        $this->lintManager = new NullLintManager();
```

Comment:
I know what you mean. Just see no big benefits for passing them as args rather than calling setters.
And we do cal setters in all other classes, so I think we can keep it that way.